subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_iso.c

```diff
@@ -746,20 +751,48 @@ static void isr_tx_common(void *param,
 						 lll->phy, lll->phy_flags);
 
 		radio_isr_set(isr_tx, lll);
-	}
 
-	if (IS_ENABLED(CONFIG_BT_CTLR_PROFILE_ISR)) {
-		/* setup capture of PDU end timestamp */
-		radio_tmr_end_capture();
+#if defined(HAL_RADIO_GPIO_HAVE_PA_PIN)
+		/* local variable used later to store iss_us next subevent PA
+		 * setup.
+		 */
+		pa_iss_us = iss_us;
+#endif /* HAL_RADIO_GPIO_HAVE_PA_PIN */
 	}
 
-	/* assert if radio packet ptr is not set and radio started rx */
+	/* assert if radio packet ptr is not set and radio started tx */
 	LL_ASSERT(!radio_is_ready());
 
 	if (IS_ENABLED(CONFIG_BT_CTLR_PROFILE_ISR)) {
 		lll_prof_cputime_capture();
 	}
 
+#if defined(CONFIG_BT_CTLR_PROFILE_ISR) || defined(HAL_RADIO_GPIO_HAVE_PA_PIN)
+	/* setup capture of PDU end timestamp */
+	radio_tmr_end_capture();
+#endif /* CONFIG_BT_CTLR_PROFILE_ISR || HAL_RADIO_GPIO_HAVE_PA_PIN */
+
+#if defined(HAL_RADIO_GPIO_HAVE_PA_PIN)
```

Comment:
Ditto with `IS_ENABLED`

Reply:
Here local `pa_iss_us` variable is conditionally defined, hence can not use `IS_ENABLED`.

Reply:
Is it important that `pa_iss_us` is assigned before line 775? Otherwise it seems you can move lines 475-477 and 755-760 into this block here and have it something like 

```c
if (IS_ENABLED(HAL_RADIO_GPIO_HAVE_PA_PIN)) {
	uint16_t pa_iss_us = iss_us;
	
	...
}
```